components/monitoring/grafana/base/grafana-operator.yaml

```diff
@@ -7,12 +7,12 @@ metadata:
   name: grafana-operator
   namespace: appstudio-grafana
 spec:
-  channel: v4
+  channel: v5
   installPlanApproval: Automatic
   name: grafana-operator
   source: community-operators
   sourceNamespace: openshift-marketplace
-  startingCSV: grafana-operator.v4.10.0
+  startingCSV: grafana-operator.v5.3.0
```

Comment:
I would avoid setting this, in the past I could see that for new installations the stratingCSV version was installed and then updated to latest. So it's better to install newest version from the channel.

Reply:
I removed it